docs/sources/enterprise/usage-insights/export-logs.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Export logs of usage insights"
+description = "Export logs of usage insights"
+keywords = ["grafana", "export", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/export-logs.md"]
+weight = 500
++++
+
+# Export logs of usage insights
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+By exporting usage logs to Loki, you can directly query them and create dashboards of the information that matters to you most, such as dashboard errors, most active organizations, or your top-10 most-used queries. Currently, usage logs can only be exported to Loki.
+
+## Usage insights logs
+
+Usage insights logs are JSON objects that represent various user activity:
```

Comment:
```suggestion
Usage insights logs are JSON objects that represent certain user activities:
```

Reply:
Rephased from (in this case) an uncountable noun to a countable noun because the items that follow are countable events.